web-services/query/src/main/java/datawave/webservice/query/logic/composite/CompositeQueryLogic.java

```diff
@@ -142,19 +142,17 @@ public void run() {
 
                             // special logic to deal with intermediate results
                             if (last instanceof EventBase && ((EventBase) last).isIntermediateResult()) {
-                                resetPageProcessingStartTime();
                                 // reset the page processing time to avoid getting spammed with these
+                                resetPageProcessingStartTime();
                                 // let the RunningQuery handle timeouts for long-running queries
```

Comment:
Does this comment need to be updated too?

Reply:
nope, that is exactly the comment I wanted there.